tests/common/matchers.py

```diff
@@ -32,15 +32,24 @@
 from pyramid import httpexceptions
 
 
-class any_callable(object):  # noqa: N801
+class Matcher(object):
+
+    def __eq__(self, other):
+        raise NotImplementedError('subclasses should provide an __eq__ method')
+
+    def __ne__(self, other):
+        return not self.__eq__(other)
```

Comment:
It's just to make it explicit that it's returning the complement of a function you've defined. I suppose it would probably also reduce the risk of introducing a bug if you somehow managed to define an intransitive equality operator.